### ISSUE TITLE
Encouter powershell-version problem

### ISSUE THREAD

**Issue by DatTruongManh** [MrPleasant-exe/Destiny-2-Solo-lobby#7]
The .exe execution of the script works totally fine but when I run the .ps1 script, it returns the output as shown in the photo. I have look up on Google for similar solution but none of them useful for my circumstance. Hope you can provide any solution for this.

My OS is Windows 10 Pro, if that helps anything.
![d2soloscript_firewall_problem](https://user-images.githubusercontent.com/76830552/103965213-d293b200-518f-11eb-81c0-5d88d3ac9e7c.png)

**Comment by Pleasant-exe** [comment]
Are you running as Admin?

**Comment by DatTruongManh** [comment]
Hi MrPleasant,
Sorry for my impoliteness for such a late response.
The script was later run with administrator privilege and the result return was the same.
After researching I discovered the cause was something to do with WMI repositories being corrupted/damaged, so by running the Windows Repair tool from Tweaking (selecting the Repair WMI), the issue was resolved.
Again, sorry for such a late response.
Sincerely,
DatTruongManh
